modin/core/storage_formats/pandas/parsers.py

```diff
@@ -653,9 +674,49 @@ def parse(fname, **kwargs):
         columns = [c for c in columns if c not in df.index.names and c in df.columns]
         if columns is not None:
             df = df[columns]
-        # Append the length of the index here to build it externally
```

Comment:
Why was this removed?

Reply:
I don't remember, but I think this entire function needs to get removed either way.